docs/tutorials/lut.rst

```diff
@@ -59,35 +59,35 @@ In all cases we will make use of the :mod:`QMigrate.io` module, so let's import
 
 ::
 
-    import QMigrate.io as qio
+    from QMigrate.io import read_stations, read_vmodel
 
-    stations = qio.stations("/path/to/station_file")
+    stations = read_stations("/path/to/station_file")
 
 Homogeneous velocity model
 ##########################
 Simply calculates the straight line traveltimes between stations and points in the grid.
 
 ::
 
-	qlut.compute(lut, stations, method="homogeneous", vp=5000., vs=3000.)
+	create_lut.compute(lut, stations, method="homogeneous", vp=5000., vs=3000.)
```

Comment:
In the other example scripts it's currently:

`from QMigrate.lut import compute, LUT`

`compute(lut, stations, method="homogeneous", vp=vp, vs=vs)`

I haven't thought about which is better, but should be consistent